public/css/main.css

```diff
@@ -12,16 +12,30 @@ h5,
 ul,
 p,
 ol,
+table,
+tr,
+td,
 li {
   margin: 0;
   padding: 0;
 }
 
+table {
+  border-collapse: collapse;
+  border-spacing: 0;
+}
+
 :root {
   --size-base: 16px;
   --main-sidebar-width: 16rem;
   --border-radius-base: 0.25rem;
 
+  /* -- Layers ------------------------------------------------------------- */
+  --layer-base: 1;
+  --layer-modal-overlay: 99;
+  --layer-modal: 100;
+  --layer-modal-above: 110;
```

Comment:
This needs more work, but is better than using the numbers directly which quickly gets out of control.